source/features/list-prs-for-file.tsx

```diff
@@ -155,7 +155,12 @@ async function initEditing(): Promise<void> {
 
 	const [prNumber] = prs; // First one or only one
 
-	(await elementReady('.file'))!.after(
+	const file = await elementReady('.file');
+	if (!file && pageDetect.isBlank()) {
```

Comment:
This feature does not await DOM ready, so it cannot use `exclude`. This logic somewhat matches the "global blankslate catcher" pattern